<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncryptPopover</class>
 <widget class="QWidget" name="EncryptPopover">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tTitleLabel" name="titleLabel">
     <property name="text">
      <string>Disk Encryption</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QWidget" name="mainWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="4" column="1">
       <widget class="QLineEdit" name="confirmPasswordBox">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="tStatusFrame" name="warningFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ENCRYPTION PASSWORD</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Choose a good password and remember it.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="acceptButton">
        <property name="text">
         <string>Encrypt this Installation</string>
        </property>
        <property name="icon">
         <iconset theme="dialog-ok">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="passwordBox">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Confirm Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Anyone using this device will need to know this password to start it.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
   <slots>
    <signal>backButtonClicked()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>tStatusFrame</class>
   <extends>QFrame</extends>
   <header location="global">tstatusframe.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>passwordBox</tabstop>
  <tabstop>confirmPasswordBox</tabstop>
  <tabstop>acceptButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>passwordBox</sender>
   <signal>returnPressed()</signal>
   <receiver>confirmPasswordBox</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirmPasswordBox</sender>
   <signal>returnPressed()</signal>
   <receiver>acceptButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>492</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>492</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
